docs/guide/extending/extending_form_builder.md

```diff
@@ -199,13 +199,19 @@
     By adjusting them to your needs, you will change your email template.
 
     ### Configure sender details
-
-    To send emails you also need to configure `sender_address` in `app/config/config.yml`.
-    It acts as a sender address and a return address where all bounced messages will be returned to.
-    You can learn more by visiting [Symfony Mailer Configuration Reference.](https://symfony.com/doc/3.4/reference/configuration/swiftmailer.html#sender-address)
-
-    ### Configure email notifications
-
-    To send emails, you need to configure `sender_address` in `app/config/config.yml` under the `swiftmailer` key.
-    It acts as a sender and return address for all bounced messages.
-    For details, see [Symfony Mailer Configuration Reference.](https://symfony.com/doc/3.4/reference/configuration/swiftmailer.html#sender-address)
+    
+    Some e-mail providers require a sender address to be set, so to avoid unsent emails when using From Builder, it is recommended to configure `sender_address` in `app/config/config.yml`. It acts as a sender and return address for all bounced messages. For more information, see [Symfony Mailer Configuration Reference.](https://symfony.com/doc/3.4/reference/configuration/swiftmailer.html#sender-address)
```

Comment:
```suggestion
    Some email providers require a sender address to be set, so to avoid unsent emails when using Form Builder,
    it is recommended to configure `sender_address` in `app/config/config.yml`.
    This email acts as a sender and return address for all bounced messages.
    For more information, see [Symfony Mailer Configuration Reference.](https://symfony.com/doc/3.4/reference/configuration/swiftmailer.html#sender-address)
```